the-experts-fe/src/components/ContractorProfile/ContractorProfile.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Contractor from "../contractor/contractor";
+import "./ContractorProfile.css";
+
+export default function ContractorProfile(props) {
+  
+  const contractor = props.location.state.contractor;
+  console.log(contractor);
+  
+  const ContractorProfile = () => {
```

Comment:
You've got a function called `ContractorProfile` inside another function called `ContractorProfile`...not a good idea! This could lead to all sorts of weird side-effects!

Also, you're fetching data and then doing nothing with it...?